composer.json

```diff
@@ -16,8 +16,6 @@
     "minimum-stability": "dev",
     "require-dev": {
         "symfony/symfony": "2.6",
```

Comment:
Yeah I think you are right, we should remove this and restore the things I deleted below. But the actual issue is something general to composer atm I think.